### PR TITLE
 - Update the TextView with the results from the sesnor.

### PR DESCRIPTION
**Pull request by vintummala**
- Fix the Logs.
 - Add README to the Samples missing README File.
 - Add TextView or the layout folder if it is missing in the Sample.

Signed-off-by: Vineela Tummalapalli <vineela.tummalapalli@intel.com>

### CODE REVIEW

File: ads1015/src/main/java/com/example/upm/androidthings/driversamples/ADCActivity.java [hunk 1]
```diff
@@ -88,14 +93,15 @@ public void run() {
             Executors.newSingleThreadScheduledExecutor().schedule(new Runnable() {
```
Comment (bjbeare) [inline ads1015/src/main/java/com/example/upm/androidthings/driversamples/ADCActivity.java]: Why all of the complexity to set running to false after a few seconds?

File: ssd1351/src/main/java/com/example/upm/androidthings/driversamples/SpiDisplayActivity.java [hunk 2]
```diff
@@ -106,6 +114,7 @@ public void run() {
                 display.setCursor((short)5, (short)70);
                 display.print("IoTDK");
                 display.refresh();
+                updateUI("Intel IoTDK");
```
Comment (bjbeare) [inline ssd1351/src/main/java/com/example/upm/androidthings/driversamples/SpiDisplayActivity.java]: same message as above. Change to "IoTDK" without "Intel" as in the display.print?

File: ads1015/src/main/java/com/example/upm/androidthings/driversamples/ADCActivity.java (hunk 1 above)
Comment (vintummala) [inline ads1015/src/main/java/com/example/upm/androidthings/driversamples/ADCActivity.java]: In android handler.postdelayed() is used to execute a command after some specified time and in java they were using Executers to execute a command after specific time. I just used whatever is there in the java examples.

File: ssd1351/src/main/java/com/example/upm/androidthings/driversamples/SpiDisplayActivity.java (hunk 2 above)
Comment (vintummala) [inline ssd1351/src/main/java/com/example/upm/androidthings/driversamples/SpiDisplayActivity.java]: In the Sample, the SPI display even prints "Intel" and then prints "IoTDK". i just concatinated them both in the Text View :). I can just add "IoTDK" instead.

**Comment by bjbeare** [comment]
something to show that it is complete would be good.

**Comment by bjbeare** [comment]
I suggest that you drop that and just do some number of iterations with a delay between them.
We aren't planning on how to show different framework features.

**Comment by vintummala** [comment]
@bjbeare I used Android studio's code formatter and done the formating and fixed indentation, white spaces etc. Please review it

File: ads1015/src/main/java/com/example/upm/androidthings/driversamples/ADCActivity.java [hunk 3]
```diff
@@ -1,39 +1,97 @@
...
+                    id++;
+                    Thread.sleep(1000);
+                }
+
+            } catch (IOException e) {
+                Log.e(TAG, "Failed to open output file for writing: ", e);
+            } catch (InterruptedException e) {
+                Thread.currentThread().interrupt();
+            } finally {
+                if (i >= 10) {
```
Comment (bjbeare) [inline ads1015/src/main/java/com/example/upm/androidthings/driversamples/ADCActivity.java]: This test for i>=10 ... what purpose does it serve?
How about:
if (bw) ...
if (fw) ...

**Comment by vintummala** [comment]
agreed :). Fixed it.

**Pull request merged**